app/src/main/java/com/sunasterisk/boringweather/ui/detail/DetailFragment.kt

```diff
@@ -83,4 +83,16 @@ class DetailFragment : BaseFragment() {
             imageBackground.translationY = (1 - scrollProgress * actionBarSize) * imageScaling
         }
     }
+
+    companion object {
+        fun newInstance(cityId: Int, dailyWeatherDateTime: Long) = DetailFragment().apply {
+            arguments = Bundle().apply {
+                putInt(ARGUMENT_CITY_ID, cityId)
+                putLong(ARGUMENT_DAILY_WEATHER_DATE_TIME, dailyWeatherDateTime)
+            }
+        }
+
+        private const val ARGUMENT_CITY_ID = "city_id"
+        private const val ARGUMENT_DAILY_WEATHER_DATE_TIME = "dailyWeather_dt"
```

Comment:
Thuộc tính viết trước function ông

Reply:
```suggestion
        private const val KEY_ARGUMENT_CITY_ID = "city_id"
        private const val KEY_ARGUMENT_DAILY_WEATHER_DATE_TIME = "dailyWeather_dt"
```
Ý ông là như này à? Ông có thể suggest được không